src/Http/ProxyRouter.php

```diff
@@ -328,4 +328,33 @@ public function proxify(): void
         header(sprintf('Content-Length: %s', filesize($target_file)));
         readfile($target_file);
     }
+
+    public function handleRedirects()
+    {
+        $this->handleWellKnownURIs();
```

Comment:
You have to exit somewhere, either here, either in the `handleWellKnownURIs` method, to prevent the following router methods to be executed.
```suggestion
        if ($this->handleWellKnownURIs()) {
            exit();
        }
```